docs/src/modules/components/MarkdownDocs.js

```diff
@@ -46,6 +46,7 @@ function MarkdownDocs(props) {
   const {
     classes,
     disableAd,
+    disableTocs,
```

Comment:
`disableToc`

Reply:
Table of Contents?

Reply:
A Table of Contents is singular – there's one "table" (list). TOCs would be multiple Table of Contents'.

Reply:
It looks like it's abbreviated to TOC anyway https://en.wikipedia.org/wiki/Table_of_contents

Reply:
🤷🏻‍♂️What did I just say?

Reply:
Haha, I would call it TOC anyway just was curious :P

Reply:
Ok thanks, I'm gonna change it.